admin/vagrant-acceptance-targets/fedora-20/Vagrantfile

```diff
@@ -10,6 +10,329 @@ VAGRANTFILE_API_VERSION = "2"
 
 ENV['VAGRANT_DEFAULT_PROVIDER'] = 'virtualbox'
 
+clustercrt = <<EOL
```

Comment:
These seem like duplicates of the ones that'll be in admin. Instead of copy/pasting them, read them in.